packages/playground/src/weblets/profile_manager.vue

```diff
@@ -185,6 +185,7 @@
                   :rules="[
                     validators.required('Password is required.'),
                     validators.minLength('Password must be at least 6 characters.', 6),
+                    validators.isAlphanumeric('Password should consist of letters and numbers only.'),
```

Comment:
not sure if we should change the rules of the password of the wallet connector as it's frequently used by the user and that may prevent the login for them

Reply:
I reverted the previous changes and verified on white spaces only. should I remove the changes from the profile manager?

Reply:
yes, it is used only for encrypting the mnemonic in local storage and works fine with spaces

Reply:
Done.